lib/libc/stdlib/malloc/mrs/mrs.c

```diff
@@ -719,6 +719,9 @@ static inline void quarantine_flush(struct mrs_quarantine *quarantine) {
 	}
 #endif /* !JUST_QUARANTINE && !JUST_PAINT_BITMAP */
 
+	if (quarantine->list == NULL)
+		return;
```

Comment:
This loses the mrs_debug_printf at the end of the function. Can we just guard the problematic few lines involving prev with a null check instead?